exercises/shared/.docs/help.md

```diff
@@ -0,0 +1,3 @@
+# Help
+
+If you're having trouble, feel free to ask help in the [Exercism support gitter channel](https://gitter.im/exercism/support).
```

Comment:
```suggestion
```

Reply:
> I don't know if there is any Julia-specific way to get help.

I'm somewhat hesitant to point students at non-Exercism channels, so I guess it should remain empty for now.

Reply:
Do you know if it's valid for this file to be entirely empty?

Reply:
I think it's valid. Configlet doesn't complain so far about the file being empty, and the only side effect will be that, in the `HELP.md` file downloaded to the student's computer, they will get an extra empty line after the general help text 🤷 

https://github.com/exercism/website/blob/baca4c21e753a2ca56c156790ffc958c319c7299/app/commands/solution/generate_help_file.rb#L45-L61